### PR TITLE
uvloop v0.20.0 - including latest `c_stdlib`

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fixes https://github.com/conda-forge/uvloop-feedstock/pull/45

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython <3                              # [build_platform != target_platform]
-    - sysroot_{{ target_platform }} 2.17     # [linux and cdt_name == "cos7"]
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```
Comment (jakirkham) [inline recipe/meta.yaml]: nit: Let's match quotes here. Also slight preference for adding `stdlib` first

```suggestion
    - {{ stdlib("c") }}
    - {{ compiler("c") }}
```

**Review by jakirkham — APPROVED** [review]
Thanks Jan! 🙏

**Comment by jan-janssen** [comment]
> Thanks Jan! 🙏

Sure - I am happy to help. I maintain https://github.com/conda-forge/graphrag-feedstock which now depends on `uvloop`. So if you need help maintaining `uvloop`, I am happy to join.

**Pull request merged**